### ISSUE TITLE
double slashes in jsx

### ISSUE THREAD

**Issue by davis** [babel/babel-sublime#186]
![screen shot 2015-10-06 at 4 12 39 pm](https://cloud.githubusercontent.com/assets/4855619/10325304/185c1722-6c45-11e5-800d-669285fe9ace.png)

**Comment by davis** [comment]
notice on line 6 i have double slashes then after that the syntax highlighting breaks

**Comment by davis** [comment]
hmm it might actually have something to do with sublime linter...
```
SublimeLinter: Traceback (most recent call last):
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 58, in loop
    item = self.q.get(block=True, timeout=self.MIN_DELAY)
  File "./queue.py", line 175, in get
queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 66, in loop
    self.lint(view_id, timestamp)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 112, in lint
    self.callback(view_id, timestamp)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/sublimelinter.py", line 120, in lint
    Linter.lint_view(view, filename, code, hit_time, callback)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/linter.py", line 919, in lint_view
    linter.lint(hit_time)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/node_linter.py", line 78, in lint
    super(NodeLinter, self).lint(hit_time)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/linter.py", line 1379, in lint
    output = self.run(cmd, self.code)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/linter.py", line 1785, in run
    return self.communicate(cmd, code)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter-contrib-eslint/linter.py", line 89, in communicate
    relfilename = os.path.relpath(self.filename, vars['folder'])
  File "./posixpath.py", line 456, in relpath
ValueError: no path specified
 
SublimeLinter: -------------------- 
SublimeLinter: chdir not set or invalid, using /Users/davis/Library/Application Support/Sublime Text 3/Packages 
SublimeLinter: eslint: <unsaved> ['/usr/local/bin/eslint', '--format', 'compact', '--stdin', '--stdin-filename', '__RELATIVE_TO_FOLDER__'] 
SublimeLinter: error in SublimeLinter daemon: 
SublimeLinter: -------------------- 
SublimeLinter: Traceback (most recent call last):
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 58, in loop
    item = self.q.get(block=True, timeout=self.MIN_DELAY)
  File "./queue.py", line 175, in get
queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 66, in loop
    self.lint(view_id, timestamp)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 112, in lint
    self.callback(view_id, timestamp)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/sublimelinter.py", line 120, in lint
    Linter.lint_view(view, filename, code, hit_time, callback)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/linter.py", line 919, in lint_view
    linter.lint(hit_time)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/node_linter.py", line 78, in lint
    super(NodeLinter, self).lint(hit_time)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/linter.py", line 1379, in lint
    output = self.run(cmd, self.code)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/linter.py", line 1785, in run
    return self.communicate(cmd, code)
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter-contrib-eslint/linter.py", line 89, in communicate
    relfilename = os.path.relpath(self.filename, vars['folder'])
  File "./posixpath.py", line 456, in relpath
ValueError: no path specified
 
SublimeLinter: -------------------- 
SublimeLinter: chdir not set or invalid, using /Users/davis/Library/Application Support/Sublime Text 3/Packages 
SublimeLinter: eslint: <unsaved> ['/usr/local/bin/eslint', '--format', 'compact', '--stdin', '--stdin-filename', '__RELATIVE_TO_FOLDER__'] 
SublimeLinter: error in SublimeLinter daemon: 
SublimeLinter: -------------------- 
SublimeLinter: Traceback (most recent call last):
  File "/Users/davis/Library/Application Support/Sublime Text 3/Packages/SublimeLinter/lint/queue.py", line 58, in loop
    item = self.q.get(block=True, timeout=self.MIN_DELAY)
  File "./queue.py", line 175, in get
queue.Empty
```

**Comment by zertosh** [comment]
Ah yeah, my fault - I broke that in #184 but only for users using ST3 dev. I have a few more fixes to push out, I'll get those out soon.

**Comment by davis** [comment]
ah, got it. thanks for the fast response!

**Comment by zertosh** [comment]
The fix is in v8.4.0.
